frontend/src/types.ts

```diff
@@ -303,7 +303,7 @@ export type ConsoleLinkKind = {
     text: string;
     location: string;
     href: string;
-    applicationMenu: {
+    applicationMenu?: {
       section: string;
       imageURL: string;
     };
```

Comment:
nit: This type is using the deprecated `k8sResrouceCommon` definition, we might wanna change it to the current one located in `k8sTypes.ts`. In addition to that, this type resource seems to fit the requirement to be placed in `k8sTypes.ts` we might wanna move it there.